app/src/main/java/org/simple/clinic/editpatient/EditPatientModel.kt

```diff
@@ -19,7 +20,8 @@ data class EditPatientModel(
     val savedPatient: Patient,
     val savedAddress: PatientAddress,
     val savedPhoneNumber: PatientPhoneNumber?,
-    val savedBangladeshNationalId: BusinessId?
+    val savedBangladeshNationalId: BusinessId?,
+    val saveButtonState: ButtonState? = null
```

Comment:
Can we avoid using default values in the constructors? This should be set in the factory function.